### ISSUE TITLE
Map info window has mixed buttons and information issues

### ISSUE THREAD

**Issue by abirger** [ioos/service-monitor#317]
A little window with the info about glider's data that opens over glider map, contains a mix of working and false buttons. The latter "pretend" to be a link (cursor changes while hovering over them) but click does not do anything.

The window also reports "Last Pinged: Never", although I believe we agreed to get rid of pinging info altogether.

Example: http://104.131.92.169/map/Glider_DAC_2 , and then select the very first "OOI GP05MOAS-GL002".

**Comment by carmelortiz** [comment]
Note this issue is not specific to the glider map. 

We should change "Last Pinged" to "Last Harvested".

**Comment by kknee** [comment]
@carmelortiz that's why I put it in the Catalog Map Improvements milestone and not the Glider Milestone.  Should we edit the title?  Or close and start a more general issue?

**Comment by carmelortiz** [comment]
@kknee Right! It is a generic map UI issue, but at some point during today's call I thought we decide to fix the issue during glider-related enhancements (i.e. not wait for 3.5). Then we flip-flopped on that, so I'll put it back in 3.5.
